charts/passbolt/templates/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 {{- $fullName := include "passbolt.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
 {{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
In Kubernetes 1.14, there is no kind Ingress in this API version.

Please keep v1beta1 or add an other test on `.Capabilities.KubeVersion.GitVersion`. If you choose the last option, you also need to adapt the spec below.

Checkout this comment for details : https://github.com/kubernetes/kubernetes/issues/90077#issuecomment-718778960